extensions/rendering-hints/README.md

```diff
@@ -0,0 +1,63 @@
+# Rendering Hints Extension Specification
+
+- **Title: Rendering Hints**
+- **Identifier: rendering_hints**
+- **Field Name Prefix: render**
+- **Scope: Item**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+One of the things many people want to do with data cataloged in STAC is to render it, often doing so directly
+from the assets (usually [Cloud Optimized GeoTIFF's](http://cogeo.org)) with dynamic web tile services. To 
+perform well these services benefit from a few 'hints' that can help the renderer perform more effectively. 
+
+This document explains the fields of the STAC Rendering Hints (`render`) Extension to a STAC Item. They are meant
+to be 'hints' - things that could be discovered by inspecting the data, but provided in STAC to help out tools
+that can take advantage of them. It is written with geospatial raster data in mind, but could apply to other data types.
+
+The field names defined herein should be added as fields in the Item Properties object. When specified on an Item, 
+the values are assumed to apply to all non-thumbnail Assets in that Item. 
+
+## Examples
+
+- TODO
+
+## Schema
+
+- TODO
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| render:max_zoom  | integer | **REQUIRED.** The maximum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that corresponds with the highest resolution of the data. Specified as [Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) zoom levels. |
+| render:min_zoom  | integer  | The minimum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that tells renderers where the overviews of the data stop. |
+| render:type    | string | The data `type` (float, int, complex, etc) to let the renderer apply any needed rescaling up front. The full set of options is listed below. |
```

Comment:
I think `dtype` would be more explicit.

Reply:
I might go full-on data_type in that case (as I'm guessing that's what the d is, but not sure, which is why I'd like to make it explicit). I was going back and forth between the two.

Reply:
`dtype` is pretty common in the Python world (that's the term Numpy and Pandas use) but I think `data_type` would be most clear to non-Python users.

Reply:
Cool. I think the audience for this is beyond python users, so will go for data_type.